include/alpaka/core/BoostPredef.hpp

```diff
@@ -11,6 +11,11 @@
 
 #include <boost/predef.h>
 
+#ifdef __INTEL_COMPILER
+#    error                                                                                                            \
+        "Support for the Intel Classic compiler (icpc) has stopped. Please upgrade to the Intel LLVM compiler (ipcx)."
```

Comment:
```suggestion
        "The Intel Classic compiler (icpc) is no longer supported. Please upgrade to the Intel LLVM compiler (ipcx)."
```

Also again: warning or error?

Reply:
warning sounds good.